packages/manager/apps/pci-private-registry/src/pages/create/steps/PlanStep.tsx

```diff
@@ -0,0 +1,158 @@
+import { OsdsButton, OsdsText } from '@ovhcloud/ods-components/react';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  StepComponent,
+  TilesInputComponent,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { useTranslation } from 'react-i18next';
+import { useContext } from 'react';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { useNavigate, useParams } from 'react-router-dom';
+import PlanComponent from '@/pages/create/Plan.component';
+import { StepEnum } from '@/pages/create/types';
+import { useStore } from '@/pages/create/store';
+import { useGetCapabilities } from '@/api/hooks/useCapabilities';
+import { PRIVATE_REGISTRY_CREATE_PLAN } from '@/pages/create/constants';
+
+export default function PlanStep(): JSX.Element {
+  const { t: tCreate } = useTranslation('create');
+  const { t: tCommon } = useTranslation('common');
+  const { t: tUpgrade } = useTranslation('upgrade');
+
+  const { tracking } = useContext(ShellContext)?.shell || {};
+
+  const { projectId } = useParams();
+  const navigate = useNavigate();
+
+  const store = useStore();
+
+  const { addSuccess, addError } = useNotifications();
+
+  const { data: capabilities } = useGetCapabilities(projectId);
+
+  const on = {
```

Comment:
```suggestion
  const creationCallbacks = {
```

for readability i would suggest avoiding "on" and replacing it, creationCallbacks is a suggestion i'm open to other proposal